egs/csj/align1/run.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+
+# CTC segmentation example recipe
+
+# Copyright 2017, 2020 Johns Hopkins University (Shinji Watanabe, Xuankai Chang)
+# 2020, Technische Universität München, Authors: Dominik Winkelbauer, Ludwig Kürzinger
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+
+# general configuration
+python=python3
+stage=-1       # start from -1 if you need to start from data download
+stop_stage=100
+ngpu=1         # number of gpus ("0" uses cpu, otherwise use gpu)
+debugmode=1
+dumpdir=dump   # directory to dump full features
+verbose=1      # verbose option
+
+# feature configuration
+do_delta=false
+cmvn=
+
+# data
+CSJDATATOP=/export/corpora5/CSJ/USB
+CSJVER=usb                          # see kaldi/egs/csj/s5/run.sh about the version
+
+# exp tag
+tag="" # tag for managing experiments.
+
```

Comment:
```suggestion
```
According to https://github.com/espnet/espnet/pull/2531/checks?check_run_id=1185652786#step:7:211